docs/guides/microsoft-graph.md

```diff
@@ -5,11 +5,11 @@ subcategory: "Upgrade Guides"
 
```

Comment:
should the name of this file include 2.0/guide-upgrade-2.0-msgraph?

Reply:
I asked about renaming docs files in the Registry, but we don't yet have a way to issue redirects and this doc already got indexed and has quite a good SEO ranking :/

Reply:
This also seems to be ranking quite well based on the page title, so in the absence of 301 support I'm inclined to leave it for now